src/commands.rs

```diff
@@ -43,25 +43,17 @@ impl Commands {
             .filter(|segment| segment.len() > 0)
             .enumerate()
             .for_each(|(i, segment)| {
-                if segment.starts_with("[") && segment.ends_with("]") {
+                if segment.starts_with("```\n") && segment.ends_with("\n```") {
```

Comment:
You could theoretically have a codeblock that ends with the triple-tick on the same line. Probably not how 99% of people are going to type it, but it's an edge case.

Reply:
Ive tested this on discord, it doesnt syntax highlight when you dont have the ` ``` ` on the next line.  But you are correct

Reply:
> This should be the last thing, unless we want to support trailing text for comments?

I believe we already have that support